.travis.yml

```diff
@@ -33,14 +33,15 @@ jobs:
     - stage: Integration Tests
       name: "Browser Tests"
       script: make run_browser_tests
-    - stage: Deploy Storybook
-      name: "Deploy Storybook"
+    - stage: Build Storybook
+      name: "Build Storybook"
       script:
         - make build_client_source
         - mkdir storybook-static
         - docker run --name client_storybook__$IMAGE_TAG libero/reviewer_client_webpack:$IMAGE_TAG
         - docker cp client_storybook__$IMAGE_TAG:/app/storybook-static/. storybook-static/
         - docker rm client_storybook__$IMAGE_TAG
+        - docker run --rm -e AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID -e AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY -v $(pwd)/storybook-static:/aws mikesir87/aws-cli aws s3 sync . s3://unstable-reviewer-storybook/$TRAVIS_PULL_REQUEST_BRANCH
```

Comment:
https://docs.travis-ci.com/user/environment-variables says `TRAVIS_PULL_REQUEST_BRANCH` is empty on `master` so it would break there. Needs a default value to point to a `master` subfolder or similar?